aptos-move/framework/aptos-stdlib/sources/string_utils.move

```diff
@@ -35,8 +35,9 @@ module aptos_std::string_utils {
     public native fun format<T>(s: &T): String;
 
     // Format a list of move values as a human readable string.
-    // eg. format_list(&std::string::utf8(b"a = {} b = {} c = {}"), &cons(1, cons(2, cons(3, nil())))) == "a = 1 b = 2 c = 3"
-    public native fun format_list<T>(fmt: &String, val: &T): String;
+    // eg. format_list(b"a = {} b = {} c = {}", &cons(1, cons(2, cons(3, nil())))) == "a = 1 b = 2 c = 3"
```

Comment:
`///` please, and when you have inline examples, use markdown with `-quotes for code examples.